en/flight_controller/nxp_rddrone_fmuk66.md

```diff
@@ -0,0 +1,119 @@
+NXP provides the schematics, gerbers, BOM and source files so that anyone can duplicate, change or repurpose this design.
+
+This is the official FMU for use with [HoverGames](https://www.hovergames.com/).
+
+
+<!--- ![RDDRONE-FMUK66 FMU Hero Image](../../assets/flight_controller/nxp_rddrone_fmuk66/nxp_rddrone_fmuk66_bottom.jpg)
+-->
+
+![RDDRONE-FMUK66 FMU Hero Image1](../../assets/flight_controller/nxp_rddrone_fmuk66/HoverGamesDrone_14042019_XL_020.jpg)
+
+![RDDRONE-FMUK66 FMU Hero Image2](../../assets/flight_controller/nxp_rddrone_fmuk66/HoverGamesDrone_14042019_XL_021.jpg)
+
+The NXP FMU and included peripherals have been tested to comply with FCC/CE/RoHs/REACH directives
+
+
+## Quick Summary
+
+- **Main FMU Processor:** 
+  - Kinetis K66 MK66FN2MOVLQ18 microcontroller running at 180MHz Cortex-M4F MCU, 2MB Flash, 256KB SRAM, Dual USBs (FS + HS), Ethernet, 144-LQFP.
+- **On-board sensors:**
+  - Accel/Gyro: BMI088/ICM42688 (RevD)...
+  - Accel/Magnetometer: FXOS8700CQ
+  - Gyro: FXAS21002CQ
+  - Magnetometer: BMM150
+  - Barometer ML3115A2
+  - Barometer BMP280
+- **GPS:**
+  - ublox Neo-M8N GPS/GLONASS receiver; integrated magnetometer IST8310
+
+
+This FMU is provided only as a kit, and includes Segger Jlink EDU mini debugger, DCD-LZ debugger adapter, USB-TTL-3V3 console cable, HolyBro GPS module, battery power module, SDCard and case, screws and stickers.
+Telemetry radios (HGD-TELEM433 and HGD-TELEM915) must be purchased separately to match ISM band frequencies used in your country.
+
+![RDDRONE-FMUK66 FMU Kit](../../assets/flight_controller/nxp_rddrone_fmuk66/RDDRONE-FMUK66-KIT-IMG.webp)
+
+A "Lite" version RDDRONE-FMUK66L is also available which does not include the power module, GPS, Jlink or USB-TTL-3V3 console cable or SDCard.
+
+Additional information can be found in the [Technical Data Sheet](https://www.nxp.com/design/designs/px4-robotic-drone-fmu-rddrone-fmuk66:RDDRONE-FMUK66). <!-- www.nxp.com/rddrone-fmuk66 -->
+
+
+## Purchase
+
+**RDDRONE-FMUK66** reference design kit may be purchased direct from NXP or from any of NXP's authorised worldwide network of [electronics distributors](https://www.nxp.com/support/sample-and-buy/distributor-network:DISTRIBUTORS).
+
+- [Purchase Link](https://www.nxp.com/design/designs/px4-robotic-drone-fmu-rddrone-fmuk66:RDDRONE-FMUK66#buy) (www.nxp.com)
+- Telemetry radios are purchased separately depending on frequency band:
+  - [HGD-TELEM433](https://www.nxp.com/part/HGD-TELEM433)
+  - [HGD-TELEM915](https://www.nxp.com/part/HGD-TELEM915)
+
+
+> **Note** *RDDRONE-FMUK66* FMU is also included in the complete HoverGames drone kit: [KIT-HGDRONEK66](https://www.nxp.com/applications/solutions/industrial/aerospace-and-mobile-robotics/uavs-drones-and-rovers/nxp-hovergames-drone-kit-including-rddrone-fmuk66-and-peripherals:KIT-HGDRONEK66#buy)
+
+
+
+## Connectors
+
+[Connector Diagram]
+
+## Pinouts
+
+[Pinouts listing or link]
```

Comment:
@igalloway This is a placholder - either please include info, or link to your docs for this, or possibly remove if you think is all covered in your other links and that people can easily find this.